### DISCUSSION TITLE
blog/react_compiler_4/

### DISCUSSION THREAD

**Discussion by giscus** [yongsk0066/yongsk0066.github.io#5] [Comments]
# blog/react_compiler_4/

React Compiler의 SSA 변환에 대해 알아보자.

https://yongsk0066.github.io/blog/react_compiler_4/

**Comment by solomon-maeng** [comment]
따봉!

**Comment by kimjeongwonn** [comment]
쌤 너무 어려워용

**Reply by yongsk0066** [reply]
보충수업 참석 바랍니다
